### PR TITLE
REFS: #98 and fixes #98 (Day-11 to 15)

### PR DESCRIPTION
**Pull request by paresh72**

### CODE REVIEW

**Review by MilitPanchasara — COMMENTED** [review]
css/js links not working in Day11 assignment, plz check the same.

**Review by MilitPanchasara — COMMENTED** [review]
Day13 => Post Detail page not done, Change Password Modal remaining, Can't access drop down menu in navigation from Posts/Categories/Users pages, Styling not proper in edit profile page.

**Review by MilitPanchasara — COMMENTED** [review]
Day 15 => Apply Bootstrap in Assignment, also reset the Result when reset button is pressed.
Day 15 => In practice exercise, calculate total of numbers which are >40. Redo the logic.

**Review by MilitPanchasara — COMMENTED** [review]
Day14 => Many files are opening in new window  (for example: Day5,9,10), try to fit content in the same page.

File: Modules/SQL/Day1/paresh gohil/Practice_exercise_1_2_3_4.sql [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	FirstName varchar(10) NOT NULL,
+	LastName varchar(10) NOT NULL,
+	Email varchar(25) NOT NULL,
+	PhoneNumber bigint NOT NULL,
+	HireDate date NOT NULL,
+	Job_ID varchar(10) NOT NULL CONSTRAINT FKJob_ID FOREIGN KEY REFERENCES Jobs(JobID),
+	Salary decimal(8,2) DEFAULT NULL, 
+	Commission decimal(2,2) DEFAULT NULL, 
+	MangerID decimal(6,0) DEFAULT NULL, 
+	DEPARTMENT_ID varchar(20) NOT NULL CONSTRAINT FKDEPARTMENT_ID FOREIGN KEY REFERENCES  Jobhistories(DepartmentID)
```
Comment (radixdt12020) [inline Modules/SQL/Day1/paresh gohil/Practice_exercise_1_2_3_4.sql:41]: there is no department Table . Employee Table departmentid field should refer department table departmentid

**Review by radixdt12020 — CHANGES_REQUESTED** [review]
there is no department Table . Employee Table departmentid field should refer department table departmentid

**Review by radixdt12020 — APPROVED** [review]

**Pull request merged**